docs/examples/1.3.x/server-nodejs/examples/storage/create-file.md

```diff
@@ -12,10 +12,11 @@ client
     .setKey('919c2d18fb5d4...a2ae413da83346ad2') // Your secret API key
 ;
 
-const promise = storage.createFile('[BUCKET_ID]', '[FILE_ID]', InputFile.fromPath('/path/to/file.png', 'file.png'));
+const fileStream = fs.createReadStream('/path/to/file.png')
+const promise = storage.createFile('[BUCKET_ID]', '[FILE_ID]', InputFile.fromPath(fileStream, 'file.png'));
```

Comment:
`InputFile.fromPath()` takes a path, not a stream.

Are you running this code in node? Can you provide the full stack trace and/or additional detail/context?